1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -22,7 +22,11 @@ let shapes = {
 };
 
 let shapesWithMoreThanFiveSides = [];
-
+for (let keys in shapes) {
```

Comment:
Good. Sometimes it is better to use a variable to describe what dta you are working with

if (shapes[keys] >= 5) does not tell me that you are comparing the number of sides of the shape. Something like tihs would be clearer:

```
for (let shape in shapes) {
  let sides = shapes[shape];
  if (sides >= 5) {
    shapesWithMoreThanFiveSides.push(shape);
  }
}
```